tasks/Task.cpp

```diff
@@ -93,13 +93,17 @@ bool Task::configureHook()
     cVb.buildFrom(cMb);
     task.set_cVe(cVb);
 
+    //set the time for trigger the timeout
+    timeout = base::Timeout(base::Time::fromSeconds(1));
```

Comment:
> I don't agree. If we want this, then the velocity/position controllers should start controlling at zero if it they have no commands.

Good point. 

Writing zero velocities aimed to handle two situations: 1) Eventual marker losing. 2) Big detection time. For (1) we should handle it on syskit level. For (2), we can just consider that if the feedback is not fast enough for control the process, then it is a not controllable system.

Considering this, at least for visual based control we should think about to increase the `aligned_vel2body_effort` timeout that currently it is 1 second. 
Apriltags detector running at 2K resolution detects the marker at 500ms average, but some times it goes to 1.2 seconds which would raise a TIMEOUT exception. I down-scale the image in order to achieve 100ms average, but experiences with SURF detectors can easily achieve 1s.

Moreover, what do you think about make this component `port_driven` ?